docs/Displaying a User Interface/Slider/index.md

```diff
@@ -215,6 +262,17 @@ sdlManager.sendRPC(slider);
 ```
 !@
 
+@![javascript]
+```js
+const sliderResponse = await sdlManager.sendRpc(slider).catch(error => error);
```

Comment:
Catching this error like this seems odd in comparison to the below `else` of handling the error (this is done in a number of places).